doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -0,0 +1,696 @@
+         |        +--ro oc-qos-ext:size                       uint32
+         |        +--ro oc-qos-ext:static-threshold           uint32
+         |        +--ro oc-qos-ext:dynamic-threshold          uint32
+         |        +--ro oc-qos-ext:pause-threshold?           uint32
+         |        +--ro oc-qos-ext:resume-threshold?          uint32
+         |        +--ro oc-qos-ext:resume-offset-threshold?   uint32
+         +--rw oc-qos-ext:buffer-priority-groups
+         |  +--rw oc-qos-ext:buffer-priority-group* [ifname priority-group]
+         |     +--rw oc-qos-ext:ifname            -> ../config/ifname
+         |     +--rw oc-qos-ext:priority-group    -> ../config/priority-group
+         |     +--rw oc-qos-ext:config
+         |     |  +--rw oc-qos-ext:ifname?           oc-if:base-interface-ref
+         |     |  +--rw oc-qos-ext:profile           -> ../../../../buffer-profiles/buffer-profile/name
+         |        +--ro oc-qos-ext:ifname?           oc-if:base-interface-ref
+         |        +--ro oc-qos-ext:priority-group?   string
+         |        +--ro oc-qos-ext:profile           -> ../../../../buffer-profiles/buffer-profile/name
+         +--rw oc-qos-ext:buffer-queues
+            +--rw oc-qos-ext:buffer-queue* [ifname queue]
+               +--rw oc-qos-ext:ifname    -> ../config/ifname
+               +--rw oc-qos-ext:queue     -> ../config/queue
+              +--rw oc-qos-ext:config
+              |  +--rw oc-qos-ext:ifname?    oc-if:base-interface-ref
+              |  +--rw oc-qos-ext:queue?     string
+              |  +--rw oc-qos-ext:profile    -> ../../../../buffer-profiles/buffer-profile/name
+              +--ro oc-qos-ext:state
+                 +--ro oc-qos-ext:ifname?    oc-if:base-interface-ref
+                 +--ro oc-qos-ext:queue?     string
+                 +--ro oc-qos-ext:profile    -> ../../../../buffer-profiles/buffer-profile/name
+
+```
+
+### 3.6.2 CLI
+
+#### 3.6.2.1 Configuration Commands
+All commands are executed in `configuration-view`:
+```
+sonic# configure terminal
+sonic(config)#
+```
+##### 3.6.2.1.1 Initialize the buffer based on system defaults
+Initialize the buffer defaults based on platform specific values (ingress/ingress buffer pools size, buffer profile, priority-group, queue..etc)
+```
+sonic(config)# qos buffer init
+
+```
+
+##### 3.6.2.1.2 Delete buffer pool
+Clear default buffer initialization.
+```
+sonic(config)# no qos buffer init
+
+```
+##### 3.6.2.1.3 Configure buffer pool
+Configure shared head room size for the ingress buffer pool (fixed name - ingress_lossless_pool), other settings such as pool size and egress pool are automatically created during buffer init.
+Please refer user guide for platform specific defaults and based on that use these CLIs to fine-tune the numbers for the use-case.
+```
+sonic(config)# qos buffer pool ingress_lossless_pool shared-headroom-size shared-headroom-size <xoff>

```

Comment:
There is a difference in the config command and the no form of this command. Config command doesnt contain '-' and no form of the command contains '-' between 'buffer' and 'pool'. What is the correct command syntax 'buffer-pool' or 'buffer pool'? Please confirm.